pkg/gui/view_helpers.go

```diff
@@ -139,6 +139,17 @@ func (gui *Gui) returnFocus(g *gocui.Gui, v *gocui.View) error {
 	return gui.switchFocus(g, v, previousView)
 }
 
+func (gui *Gui) goToSideView(sideViewName string) func(g *gocui.Gui, v *gocui.View) error {
+	return func(g *gocui.Gui, v *gocui.View) error {
```

Comment:
could we also ensure that we close any open popup panels here? Currently we have two popup views: 'commitMessage' and 'confirmation', and we deal with each one differently. We'd need something like:
```
gui.g.SetViewOnBottom("commitMessage")
gui.closeConfirmationPrompt(gui.g)
```
It might also be worth wrapping that in a function called e.g. 'closePopupPanels' for future use in other contexts.

Sorry for the delay on the re-review btw I've been struggling to find time :)

Reply:
Good point, will have a look

Reply:
I ended up reusing the `gui.onNewPopupPanel` function, which already calls `setViewOnBottom` for 'commitMessage', 'credentials' and 'menu', and explicitly calling `gui.closeConfirmationPrompt(gui.g)`